addons/mrp/migrations/10.0.2.0/post-migration.py

```diff
@@ -278,37 +284,112 @@ def populate_stock_move_workorder_id(cr):
 
 def fill_stock_move_unit_factor(env):
```

Comment:
Took me some time to figure out what's happening here. Can you document here that the unit factor represents the rate of consumption of the raw material move lines given the produced quantities of the production order? And mention that your approach is to add temporary columns for computed fields that represent these produced quantities.